### PR TITLE
bpo-39050: The Help button in IDLE's config menu does not work

### PR DESCRIPTION
**Pull request by ZackerySpytz**
https://bugs.python.org/issue39050

### CODE REVIEW

**Comment by ZackerySpytz** [comment]
The `text` param of `view_text()`  was renamed to `contents` in 3221a63c692.

I don't think this fix needs a news entry.

**Comment by terryjreedy** [comment]
This gets a news entry, so people know that they can use the button again.  I will add one if still needed when merging.

File: Lib/idlelib/configdialog.py [hunk 1]
```diff
@@ -210,7 +210,7 @@ def help(self):
         """
         page = self.note.tab(self.note.select(), option='text').strip()
         view_text(self, title='Help for IDLE preferences',
-                 text=help_common+help_pages.get(page, ''))
+                  contents=help_common + help_pages.get(page, ''))
```
Comment (csabella) [inline Lib/idlelib/configdialog.py]: The style in IDLE for arguments is generally to not have spaces around the `+`, so no need to change this.

**Comment by csabella** [comment]
Thanks, @ZackerySpytz.  Please also add the News Entry that Terry requested.

**Review by terryjreedy — APPROVED** [review]

**Comment by terryjreedy** [comment]
I added a test that would have failed and caught the regression.  This was part of the 6% still not covered.  Zack, thanks for reporting and finding the reason.

**Pull request merged**